common/lib/xmodule/xmodule/fields.py

```diff
@@ -252,3 +253,37 @@ def enforce_type(self, value):
             return value
 
         return self.from_json(value)
+
+
+class ScoreField(JSONField):
+    """
+    Field for blocks that need to store a Score. XBlocks that implement
+    the ScorableXBlockMixin may need to store their score separately
+    from their problem state, specifically for use in staff override
+    of problem scores.
+    """
+    MUTABLE = False
+
+    def from_json(self, value):
+        if value is None:
+            return value
+        if isinstance(value, Score):
+            return value
+
+        if ('raw_earned' not in value or 'raw_possible' not in value) or any(
+            key not in ['raw_earned', 'raw_possible'] for key in value.keys()
+        ):
+            raise TypeError('Scores must contain only a raw earned and raw possible value.')
+
+        raw_earned = value['raw_earned']
+        raw_possible = value['raw_possible']
```

Comment:
Wrote that comment too quickly- I think this allows floats as-is so I just need to make sure we allow floats when creating the field.